### ISSUE TITLE
Explain to users how deterministic addresses work

### ISSUE THREAD

**Issue by PeterSurda** [mailchuck/PyBitmessage#100]
From general chan:
## 

I doubt its a bug.. ask the others if it is a bug.. I already made a backup,so if its a bug,I'll just replace it back like it was
## 

must be  bug
## 

i just made 2 new keys with new passphrase along with my already generated keys because I deleted the 2 old ones from the keys list I had before.. And it seems I still have the other old keys I generated along with the 2 new ones I generated,so nothing is gone when I created 2 new ones with a new passphrase.
## 

yes
## 

Let me rephrase my question again abit better..

When I first started BitMessage,I generated keys and it seems that the 2 old keys that I had before i generated new ones was among that list and people sent messages to these 2 old keys,so I had to delete them. After I deleted them,I had to make 2 new keys,but I had to make new passphrase for these 2 keys wich i did. 
Will the old list of keys that I made when I had to start over and the passphrase I made with it be gone since I made 2 new keys with a new passphrase to replace the 2 old ones I deleted?
## 

yes all your existing addresses and their messages are deleted
## 

I see.

What if I already have a passphrase when I first started to use BitMessage,but deleted 2 BM-keys and then make 2 new BM keys but need a different passphrase.. Won't that overwrite the old keys or will I still keep the old BM keys I generated when I first started if I make a a different passphrase?
## 

Use a different passphrase or use the random option
## 

Hi
I tried to add 2 new keys when I deleted 2,but it says something about "<BM-key> is already in "Your Identities". Not adding it again".. How do I add 2 new keys when I get that message?

**Comment by PeterSurda** [comment]
It's not really a bug, it's just badly explained or perhaps unintuitive for people who are not familiar with the concept of deterministic addresses.

**Comment by ghost** [comment]
The simplest solution is just to add a warning that this is an advanced feature, and it should only be used if you know how it works. 

It is also a dangerous feature as people can easily use weak passphrases.

**Comment by PeterSurda** [comment]
There is an alternative wizard for address creation which is half-done but not active. It includes explanations and an option to register with mailchuck. Maybe it should be just finished properly. It's under src/bitmessageqt/newaddresswizard.py

**Comment by ghost** [comment]
I just looked at the alternative wizard and I think it is way too complicated. Each new question is a chance for people to get confused or give up.

The best solution IMO is to only ask one question: "What should the label be?" Nothing about emails, passphrases, shorter addresses, or streams.
